jmbo/views.py

```diff
@@ -50,6 +50,7 @@ def get_template_names(self):
 
 class ObjectList(ListView):
     model = ModelBase
+    app_label = "jmbo"
```

Comment:
Don't make it a class level attr - just use it as the second arg to kwargs.get. 

This code looks weird in any case - it's very old, but I understand what you need to achieve. With jmbo-listing we're not using ObjectList much anymore.